common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -457,24 +457,6 @@ describe 'Problem', ->
 
         expect(window.SR.readText).toHaveBeenCalledWith 'Answers to this problem are now shown. Navigate through the problem to review it with answers inline.'
 
-      describe 'multiple choice question', ->
```

Comment:
This test (and the code associated with it) dates from 2012. But I don't see that it bares any resemblance to how we represent checkbox or multiple choice problems anymore.